src/java/fr/paris/lutece/plugins/forms/web/FormXPage.java

```diff
@@ -566,6 +567,18 @@ public synchronized XPage doFormResponseSummary( HttpServletRequest request ) th
                 return getStepView(  request );
             }
             FormsResponseUtils.fillResponseManagerWithResponses( request, true, _formResponseManager, _stepDisplayTree.getQuestions( ), false );
+            List<Step> stepList = StepHome.getStepsListByForm( form.getId( ) );
```

Comment:
We should use FormsCacheService to avoid performance problems (and keep consistency during the process)